detection_rules/kbwrap.py

```diff
@@ -147,6 +147,13 @@ def _parse_list_id(s: str):
                 )
                 click.echo()
 
+    def _matches_rule_ids(item: GenericCollectionTypes, rule_ids: dict) -> GenericCollectionTypes:
+        """Check if the item matches any of the rule IDs in the provided set."""
+        return any(
+            rule_id in rule_ids
+            for rule_id in item.contents.metadata.get("rule_ids", [])
+        )
```

Comment:
Can we use set notation?

```suggestion
    def _matches_rule_ids(item: GenericCollectionTypes, rule_ids: set) -> GenericCollectionTypes:
        """Check if the item matches any of the rule IDs in the provided set."""
        item_rule_ids = set(item.contents.metadata.get("rule_ids", []))
        return bool(item_rule_ids.intersection(rule_ids))
```